pages/devops-resources.md

```diff
@@ -110,4 +111,5 @@
 ## GitHub Repositories to learn DevOps
 |Description | Preview   | Link to get started with it   |
 | :------------: | :------------: | :------------: |
-|  1. 90 Days of DevOps - Roadmap  | <center> [![open source](https://ik.imagekit.io/1cw2zpbjy/OSWH/Getting_started_with_DevOps_-_Banner.png?ik-sdk-version=javascript-1.4.3&updatedAt=1667725617154 "DevOps")](https://github.com/MichaelCade/90DaysOfDevOps "90 Days of DevOps - Roadmap") </center> |   [Here you go](https://github.com/MichaelCade/90DaysOfDevOps)
\ No newline at end of file
+|  1. 90 Days of DevOps - Roadmap  | <center> [![open source](https://ik.imagekit.io/1cw2zpbjy/OSWH/Getting_started_with_DevOps_-_Banner.png?ik-sdk-version=javascript-1.4.3&updatedAt=1667725617154 "DevOps")](https://github.com/MichaelCade/90DaysOfDevOps "90 Days of DevOps - Roadmap") </center> |   [Here you go](https://github.com/MichaelCade/90DaysOfDevOps)
+|  2. DevOps Learning Hub: Tools and Tech  | <center> [![open source](https://ik.imagekit.io/hfclyehqd/OS_Bag/DevOps_-_Pradumnasaraf.png?updatedAt=1684307301720 "DevOps")](https://github.com/Pradumnasaraf/DevOps "DevOps Learning Hub: Tools and Tech") </center> |   [Here you go](https://github.com/Pradumnasaraf/DevOps)
```

Comment:
Keep the cover size 320 X 180 @mhdamaan79

Reply:
I have updated the preview cover with a 320x180 image ratio. Please review the changes and let me know if any further adjustments are needed.

Reply:
Sure @mhdamaan79